flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -1293,7 +1293,7 @@ private boolean restoreLatestCheckpointedStateInternal(
 				}
 			}
 
-			LOG.info("Restoring job {} from latest valid checkpoint: {}.", job, latest);
+			LOG.info("Restoring job {} from checkpoint: {}.", job, latest);
```

Comment:
I don't really see how @NicoK 's suggestion makes it clearer that it could also be a savepoint.

My suggestion would be to encode the type checkpoint/savepoint into `CompletedCheckpoint#toString` and just use: `LOG.info("Restoring job {} from  {}.", job, latest);`.

Then the full message would be 
```
8004 [flink-akka.actor.default-dispatcher-3] INFO org.apache.flink.runtime.checkpoint.CheckpointCoordinator [] - Restoring job 36681198e38a2c91932eb131f47e3641 from Checkpoint 1 @ 1597663438566 for 36681198e38a2c91932eb131f47e3641 located at file:/var/folders/dm/5xn_h6n9135dwy4j27sr65zh0000gp/T/junit492898918775365121/junit2392379173263601046/checkpoints/36681198e38a2c91932eb131f47e3641/chk-1.
```

instead of 

```
8004 [flink-akka.actor.default-dispatcher-3] INFO org.apache.flink.runtime.checkpoint.CheckpointCoordinator [] - Restoring job 36681198e38a2c91932eb131f47e3641 from checkpoint: Checkpoint 1 @ 1597663438566 for 36681198e38a2c91932eb131f47e3641 located at file:/var/folders/dm/5xn_h6n9135dwy4j27sr65zh0000gp/T/junit492898918775365121/junit2392379173263601046/checkpoints/36681198e38a2c91932eb131f47e3641/chk-1.
```

Reply:
That's a good idea!